wallet/wallet.go

```diff
@@ -288,44 +281,10 @@ func (w *Wallet) GetVoteBitsExtended() []byte {
 
 // SetTicketPurchasingEnabled is used to enable or disable ticket purchasing in the
 // wallet.
-func (w *Wallet) SetTicketPurchasingEnabled(flag bool) error {
+func (w *Wallet) SetTicketPurchasingEnabled(flag bool) {
```

Comment:
ok I see why. in the future we will want to access this info through the ticket buyer itself (if it was even started) instead of passing it through the wallet